recipe/conda_build_config.yaml

```diff
@@ -332,11 +326,9 @@ gf2x:
 gsl:
   - 2.4
 gstreamer:
-  - 1.12  # [linux]
-  - 1.14.0  # [osx]
+  - 1.14.4
 gst_plugins_base:
```

Comment:
@isuruf is it ok that it's `gst_plugins_base` here, but `gst-plugins-base` elsewhere? Looks like an inconsistency.

Reply:
That's fine. It's a conda-build thing because these keys are used as jinja variables they should be valid jinja identifiers.